recipes/zifa/meta.yaml

```diff
@@ -1,19 +1,22 @@
+{% set version = "0.1.0" %}
+{% set sha256 = "a0eccbeb52836b4bb9b1127717b78ce19b2e20177b7f34bfab8c0b5604ffc3a7" %}
+
 package:
   name: zifa
-  version: "0.1.0"
+  version: '{{version}}'
 
 source:
   url: https://github.com/epierson9/ZIFA/archive/e40fdbf6c41cd7e564ab03d242fc071f5ac543b7.zip
+  sha256: '{{sha256}}'
 
 build:
   noarch: python
-  number: 1
+  number: 2
```

Comment:
@corburn if this package wasn't build yet due to our huge backlog a increase of the build-number might be not needed. Also if you just fix linting errors, a rebuild of the package is imho also not needed, which also means not increasing the build number.

Reply:
@bgruening I'll keep that in mind.